### PR TITLE
Ocd 1599 - 2014 cms id validator

### PR DESCRIPTION
**Pull request by brianlindsy**

### CODE REVIEW

File: chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/results/CertificationIdResults.java [hunk 1]
```diff
@@ -52,8 +51,10 @@ public void setProductId(final Long id) {
     private String ehrCertificationId;
     private Map<String, Integer> metCounts;
     private Map<String, Integer> metPercentages;
-    private ArrayList<String> missingAnd;
-    private List<ArrayList<String>> missingOr;
+    protected ArrayList<String> missingAnd = new ArrayList<String>();
+    protected List<ArrayList<String>> missingOr = new ArrayList<ArrayList<String>>();
```
Comment (andlar) [inline chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/results/CertificationIdResults.java]: Checkstyle tends to complain about protected stuff. If you think it's warranted it's probably okay to leave it this way, but if you can make this private and add getters/setters it'll help with our linting counts

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/certificationId/Validator2014.java [hunk 2]
```diff
@@ -147,6 +149,13 @@ protected boolean isInpatientCqmsValid() {
                 ++cqmCount;
             }
         }
+        
+        if(!(this.counts.get("cqmsInpatientRequiredMet") >= this.counts.get("cqmsInpatientRequired"))){
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/certificationId/Validator2014.java]: I know checkstyle doesn't like `if(` but prefers `if (`

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/certificationId/Validator2014.java [hunk 3]
```diff
@@ -192,6 +219,9 @@ protected boolean isCPOEValid() {
                 return true;
             }
         }
+        ArrayList<String> missing1 = new ArrayList<String>();
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/certificationId/Validator2014.java]: Why the "1" on the end of the variable name? Why the variable at ally?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/certificationId/Validator2014.java [hunk 4]
```diff
@@ -239,7 +269,23 @@ protected boolean isTOCValid() {
             this.counts.put("criteriaTocRequired", 2);
             return true;
         }
-
+        
+        ArrayList<String> missing2 = new ArrayList<String>();
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/certificationId/Validator2014.java]: Why the "2"?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/certificationId/Validator2014.java (hunk 2 above)
Comment (tmy1313) [reply chpl/chpl-service/src/main/java/gov/healthit/chpl/certificationId/Validator2014.java]: Wouldn't this be equivalent?
`if (this.counts.get("cqmsInpatientRequiredMet") < this.counts.get("cqmsInpatientRequired")) {`

**Pull request merged**